.babelrc

```
@@ -1,6 +1,16 @@
 {
   "presets": [
-    ["env", { "targets": { "electron": "1.8.4" } }],
-    "react"
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "esmodules": true
+        }
```

non-blocking: this could target electron 8 since we don't have to worry about rendering it anywhere else. (this might reduce the bundle size too.)
```suggestion
        "targets": {
          "esmodules": true,
          "electron": 8
        }
```